xml/signals/SNCF-2015/appearance-Meca.xml

```diff
@@ -33,29 +33,26 @@
       <aspectname>Clear</aspectname>
       <show>lunar</show>
       <imagelink type="large">../../../resources/icons/SNCF/Normal/Meca/VLMecabis.gif</imagelink>
-      <imagelink type="Small">../../../resources/icons/SNCF/Small/Meca/VLMecabis.gif</imagelink>
+      <imagelink type="small">../../../resources/icons/SNCF/Small/Meca/VLMecabis.gif</imagelink>
       <imagelink type="aspects">../../../resources/icons/SNCF/Small/Meca/VLMecabis.gif</imagelink>
     </appearance>
     <appearance>
       <aspectname>Restrict30</aspectname>
       <show>yellow</show>
       <imagelink type="large">../../../resources/icons/SNCF/Normal/Meca/R30.gif</imagelink>
-      <imagelink type="Small">../../../resources/icons/SNCF/Small/Meca/R30.gif</imagelink>
+      <imagelink type="small">../../../resources/icons/SNCF/Small/Meca/R30.gif</imagelink>
```

Comment:
To keep things like "Small" creeping back in, should we have the XML Schema check that the content is one of large, small?

Reply:
Never mind - the Czech signals use lots of different values for `type`. Sorry for the confusion.

Reply:
> To keep things like "Small" creeping back in, should we have the XML Schema check that the content is one of large, small?

The `aspects.xml` defines the valid image types. If this PR gets merged, it will ensure that the image types used in the `appearance-*.xml` files are defined in the `aspects.xml` file. That's how I found this issue.